_includes/manuals/nightly/3.2.2_installer_options.md

```diff
@@ -275,10 +275,6 @@ kafo-export-params -c /etc/foreman-installer/scenarios.d/foreman.yaml -f html
         <td style='white-space:nowrap'>--foreman-app-root</td>
         <td>Name of foreman root directory</td>
       </tr>
-      <tr>
```

Comment:
We automatically generate these options. It'd be better to use the [script](https://github.com/theforeman/theforeman.org/tree/gh-pages/scripts/installer) so it matches the reality (and `--full-help`).

Reply:
removed these changes